### ISSUE TITLE
Sqitch deploy returns malloc issue

### ISSUE THREAD

**Issue by Alam080** [sqitchers/sqitch#804]
I am using Sqitch with a postgres database.

My Sqitch and Perl version are:
Perl version: v5.38.1
Sqitch version: v1.4.0

I added a new db change with Sqitch add using the below code:
`sqitch add 2023-11-27_1742_add_imported_to_card -m "add is_imported field to cards table"`

the appropriate files were created
`Created deploy/2023-11-27_1742_add_imported_to_card.sql`
`Created revert/2023-11-27_1742_add_imported_to_card.sql`
`Created verify/2023-11-27_1742_add_imported_to_card.sql`
`Added "2023-11-27_1742_add_imported_to_card" to sqitch.plan`

I then added my SQL changes and try to deploy with sqitch deploy:
`sqitch deploy dev`

But i get the following error message:
`perl(75177,0x1dd219ec0) malloc: double free for ptr 0x1448c1400`
`perl(75177,0x1dd219ec0) malloc: *** set a breakpoint in malloc_error_break to debug`
`zsh: abort  sqitch deploy dev`

Is it an issue with Sqitch code that might have been affected by Perl update?

**Comment by theory** [comment]
No, Sqitch is pure Perl, so won't cause issues like this. Most likely one of the C modules you upgraded has a new bug in it. Annoying, if unsurprising, that the error doesn't include a stack trace.
